<Type Name="ServerException" FullName="System.Runtime.Remoting.ServerException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="b9048c3cefde07a84ee8996c30ea94f505756f75" />
    <Meta Name="ms.sourcegitcommit" Value="6a0b904069161bbaec4ffd02aa7d9cf38c61e72e" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="es-ES" />
    <Meta Name="ms.lasthandoff" Value="06/24/2018" />
    <Meta Name="ms.locfileid" Value="36534261" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class ServerException : SystemException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit ServerException extends System.SystemException" />
  <TypeSignature Language="DocId" Value="T:System.Runtime.Remoting.ServerException" />
  <TypeSignature Language="VB.NET" Value="Public Class ServerException&#xA;Inherits SystemException" />
  <TypeSignature Language="C++ CLI" Value="public ref class ServerException : SystemException" />
  <TypeSignature Language="F#" Value="type ServerException = class&#xA;    inherit SystemException" />
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.SystemException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute>
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary>Excepción que se produce para comunicar errores al cliente cuando este se conecta a aplicaciones distintas de .NET Framework que no pueden producir excepciones.</summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <xref:System.Runtime.Remoting.ServerException> utiliza HRESULT COR_E_SERVER, que tiene el valor 0x8013150E.  
  
 <xref:System.Runtime.Remoting.ServerException> usa el valor predeterminado <xref:System.Object.Equals%2A> implementación, que admite la igualdad de referencia.  
  
 Para obtener una lista de valores de propiedad iniciales para una instancia de <xref:System.Runtime.Remoting.ServerException>, consulte el <xref:System.Runtime.Remoting.ServerException> constructores.  
  
 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>Inicializa una nueva instancia de la clase <see cref="T:System.Runtime.Remoting.ServerException" />.</summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ServerException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Runtime.Remoting.ServerException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ServerException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>Inicializa una nueva instancia de la clase <see cref="T:System.Runtime.Remoting.ServerException" /> con propiedades predeterminadas.</summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <xref:System.Runtime.Remoting.ServerException> hereda de la <xref:System.SystemException> clase. La siguiente tabla muestra los valores de propiedad iniciales para una instancia de la <xref:System.Runtime.Remoting.ServerException> clase:  
  
|Propiedad.|Valor|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|`null`|  
|<xref:System.Exception.Message%2A>|La cadena vacía ("")|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ServerException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Runtime.Remoting.ServerException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ServerException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.Runtime.Remoting.ServerException : string -&gt; System.Runtime.Remoting.ServerException" Usage="new System.Runtime.Remoting.ServerException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">Mensaje que describe la excepción</param>
        <summary>Inicializa una nueva instancia de la clase <see cref="T:System.Runtime.Remoting.ServerException" /> con el mensaje especificado.</summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <xref:System.Runtime.Remoting.ServerException> hereda de la <xref:System.SystemException> clase. La siguiente tabla muestra los valores de propiedad iniciales para una instancia de la <xref:System.Runtime.Remoting.ServerException> clase:  
  
|Tipo de propiedad|Condition|  
|-------------------|---------------|  
|<xref:System.Exception.InnerException%2A>|`null`|  
|<xref:System.Exception.Message%2A>|El `message` cadena|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ServerException (string message, Exception InnerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception InnerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, InnerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ServerException(System::String ^ message, Exception ^ InnerException);" />
      <MemberSignature Language="F#" Value="new System.Runtime.Remoting.ServerException : string * Exception -&gt; System.Runtime.Remoting.ServerException" Usage="new System.Runtime.Remoting.ServerException (message, InnerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="InnerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">Mensaje de error que explica la razón de la excepción.</param>
        <param name="InnerException">La excepción que es la causa de la excepción actual. Si el parámetro <c>InnerException</c> no es <see langword="null" />, se genera la excepción actual en un bloque <see langword="catch" /> que controla la excepción interna.</param>
        <summary>Inicializa una nueva instancia de la clase <see cref="T:System.Runtime.Remoting.ServerException" /> con el mensaje de error especificado y una referencia a la excepción interna que representa la causa de esta excepción.</summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 Una excepción que se produce como consecuencia directa de una excepción anterior debe incluir una referencia a esta última en la propiedad <xref:System.Exception.InnerException%2A>. La propiedad <xref:System.Exception.InnerException%2A> devuelve el mismo valor que se haya pasado al constructor, o bien `null` en caso de que la propiedad <xref:System.Exception.InnerException%2A> no le haya proporcionado al constructor el valor de la excepción interna.  
  
 La siguiente tabla muestra los valores de propiedad iniciales para una instancia de la <xref:System.Runtime.Remoting.ServerException> clase:  
  
|Propiedad.|Valor|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|La referencia a la excepción interna|  
|<xref:System.Exception.Message%2A>|La cadena de mensaje de error|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
      </Docs>
    </Member>
  </Members>
</Type>